docs/guides/integration-prebid.md

```diff
@@ -0,0 +1,187 @@
+---
```

Comment:
which page should link to this? The publisher page's resource [section](https://unifiedid.com/docs/overviews/overview-publishers#resources)?

Reply:
Yes. Done. Thx. Publisher overview and also integration guides summary.

Reply:
Publisher page and integration guides overview page. Thx.